responsive topbar/index.html

```diff
@@ -0,0 +1,362 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="topbar">
+        <input type="checkbox" id="check" />
+        <div class="logo"></div>
+        <div class="menu">
+          <a href="#">Item1</a>
+          <a href="#">Item2</a>
+          <a href="#">Item3</a>
+        </div>
+        <label for="check" class="burger"> </label>
```

Comment:
todo:
- zamiast `label` i tego `input` uzyj poprostu zwykly `button`
- zrob nowy plik `script.js` i zaimportuj go do `index.html`
- uzyj javascript zeby dac buttonowi funkcje `toggleMenu`, ktora ustawi klase na `display: none`

zasoby:
- co to w ogole jest `JS DOM` https://trello.com/c/YJx79PaA/7-tutorials **ZACZNIJ OD TEGO**
- jak dodac js do strony: https://www.digitalocean.com/community/tutorials/how-to-add-javascript-to-html
- jak dodac eventListener https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event#javascript